### PR TITLE
🎨 [Frontend] Refuse folder drop on Workbech

### PR DESCRIPTION
**Pull request by odeimaiz**
## What do these changes do?

Show error message saying that ``Folders are not accepted. You might want to upload a zip file.`` when users drop a folder on the Workbench:

RejectDroppingFolders

## Related issue/s

related to https://z43.fogbugz.com/f/cases/198533

## Dev-ops checklist

- [ ] No ENV changes or I properly updated ENV ([read the instruction](https://git.speag.com/oSparc/osparc-ops-deployment-configuration/-/blob/configs/README.md?ref_type=heads#how-to-update-env-variables))

### CODE REVIEW

**Review by pcrespov — APPROVED** [review]

File: services/static-webserver/client/source/class/osparc/workbench/WorkbenchUI.js [hunk 1]
```diff
@@ -1939,24 +1939,25 @@ qx.Class.define("osparc.workbench.WorkbenchUI", {
...
-              filePicker.uploadPendingFiles(fileList);
+              filePicker.uploadPendingFiles(files);
               filePicker.addListener("fileUploaded", () => this.fireDataEvent("nodeSelected", nodeUI.getNodeId()), this);
             }
+          } else {
+            osparc.FlashMessenger.getInstance().logAs(this.tr("Only one file at a time is accepted."), "ERROR");
           }
         } else {
-          osparc.FlashMessenger.getInstance().logAs(this.tr("Only one file is accepted"), "ERROR");
+          osparc.FlashMessenger.getInstance().logAs(this.tr("Folders are not accepted. You might want to upload a zip file."), "ERROR");
```
Comment (GitHK) [inline services/static-webserver/client/source/class/osparc/workbench/WorkbenchUI.js:1960]: Maybe a more impersonal message is better. Something like the following.
```suggestion
          osparc.FlashMessenger.getInstance().logAs(this.tr("Folders are not accepted. Upload a zip file."), "ERROR");
```

**Review by GitHK — APPROVED** [review]
👍

File: services/static-webserver/client/source/class/osparc/workbench/WorkbenchUI.js (hunk 1 above)
Comment (odeimaiz) [reply services/static-webserver/client/source/class/osparc/workbench/WorkbenchUI.js:1960]: Nah, a like the personal message, also I don't like the imperative in this case.

**Review by elisabettai — APPROVED** [review]
Nice, thanks a lot!
And I imagine the same will happen when you add the folder via the file picker (instead of direct drag-and-drop)

**Review by sanderegg — APPROVED** [review]

**Comment by odeimaiz** [comment]
> Nice, thanks a lot! And I imagine the same will happen when you add the folder via the file picker (instead of direct drag-and-drop)

You can give it a try ;)

**Review by ignapas — APPROVED** [review]

**Pull request merged**